android/src/main/java/app/shosetsu/android/view/compose/NovelCard.kt

```diff
@@ -144,18 +144,24 @@ fun NovelCardNormalContent(
 					}
 					.alpha(if (isPlaceholder) 0.0f else 1.0f)
 			)
-			Text(
-				title,
-				modifier = Modifier
+			Box(
+				Modifier
 					.align(Alignment.BottomCenter)
-					.placeholder(visible = isPlaceholder)
```

Comment:
This is meant for search views where content is loaded via paging. Why are you removing it?

Reply:
I didn't, it's still on the Text, not the Box around the Text